server/__tests__/cars.ts

```diff
@@ -92,4 +92,12 @@ describe('/cars endpoint', () => {
     expect(result.body.data.rows[0].price).toBeLessThanOrEqual(200000);
     expect(result.statusCode).toEqual(200);
   });
+  test('should return id of car that update', async () => {
+    const result = await request(app).put('/api/v1/cars/1')
+
+      .set('Cookie', 'token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYWRtaW4iLCJ1c2VySWQiOjF9.5bbTDHhlnCvs5ZDGfXcL_MU_4CCKzJ61wUDCGD_nWks');
```

Comment:
put on env and import from there, 
Also I've put it as a repo secret as `ADMIN_TOKEN`